tests/app/authentication/test_jwt_decoder.py

```diff
@@ -46,29 +46,26 @@
                  b'kwIDAQAB\n' \
                  b'-----END PUBLIC KEY-----\n'
 
-jwt = "eyJhbGciOiJub25lIn0.eyJ1c2VyIjoiamltbXkifQ."
-
-signed_jwt = "eyJhbGciOiJSUzI1NiJ9.eyJ1c2VyIjoiamltbXkifQ.oec9V6uxd9nlNsEZud9YIf1Ee" \
-             "0Su-sgVSnBPpR-8AURchaT-L_ulrDSmHRzD0BdEpuaJkbA7DgeXUsuFWpspDgtxZvomv7t" \
-             "Oel4yTKHWjZ1BzuWcZhWNMbwwzmmauUyWE7FRMVS1yFFpbpOnvuS6jOJWczVMNQEKp6PEH" \
-             "2Q99ZpJ1k9uKACNzXrILLKPSN3OJHn3sYP5Nn0WGQtol_BByE4V4YrhTSrcy3QUpKwPYcw" \
-             "4FzSchnKab0D1R9GHYHKBQaWf0f9PsbDByzF97UhZSEgJrt-kJ1fu3MCYUyrnCYzNz1W_x" \
-             "22cT5SsEiu34IKEZ2w-9XerafAEMI86FA1MOA"
-
-encrypted_jwt = "eyJjdHkiOiJKV1QiLCJlbmMiOiJBMjU2R0NNIiwiYWxnIjoiUlNBLU9BRVAifQ.D_dPbu" \
-                "r9clGwfEI1D28zRetV76GGZrU5CExTCx8T1DtZSvGrvNSwIQLR_7z4d1Cr6BIkyFvoDOh" \
-                "bgYMpPxQNBd1e24biY5EGQxphGi4vzDs6QcuEgZGLtftTnveV0rkbLzKJ6QY7V4mqbwUg" \
-                "BeEkEnEI2RmUQ9URmzlGpieyCd0hndBhG3GOEpqAw-OD7TwDIF_VNPFHva8QEHImvdTD7n" \
-                "b5dDXZ343U2iBYeVHKa3Ot6z8zKF4EL4G1t9KYASn1OTVGENjIpv7jk6pAYts5kIH9wsAm" \
-                "3KzdPmK3Evrp1ZORFkVI08BbenzQzW6rJKIxSP5awktC0s4ooofyGw6w_Q.6NEcHGWalyK" \
-                "Z0hI3.ar6axH5GB6L1K8UPNqQoJiLhwG0dZGL0Mgp_1fadQxIJXtBTmR2ocitgnzH5JxNJ" \
-                "1Z7vIxWFLU5IsEIjViUH97lXx_PjYGzW6w4ER4Tw8YXus-2wuTJUvaKnKnrZz_1TT9kn_s" \
-                "HgGmQdBfOktA-sGh_Kys9edAq7YtrnRCz8EanxWcEHGCqYeq-qT49ejpWNBJuVfaL7gH7d" \
-                "-l7XDlK-4L_RRYuf1_VhoQSCxhJyoZcTeFfrukDlQSWMmJlAEtrpBwKJe1JYSTEQVGm6Hg" \
-                "_x9nd9N_lBOmLHk8KUFCN02_Qg4PPnlTImDtbI-lmBoJroyIBC7wnQyDtJlx6rwdjKWGG6" \
-                "OCqx_1IuC1KAoSGip4SDOfnVvncennqcF6rprBWnTcOmFAl1RPZ8VoA81fZXREBCEDp91t" \
-                "ySEfPRkl1nJsWW2Mi6YYLTfsqn-lUU-W9smCRtKYTzVxwGXF2jIU-KK1c0FDJIrbzCuX-N" \
-                "2cTG8jEw-kTLf_DA11sL32YF_S_0Lpw.AN8XkD0BIi3anityXzbmhw"
+
+jwt = "eyJhbGciOiJub25lIn0.eyJleHAiOiIyMDc1Mjk3MTQ4IiwiaWF0IjoiMTQ1NDkzNzAyMyIsInVzZXIiOiJqaW1teSJ9."
```

Comment:
At the top of this file the keys are in b'str\n' format, have we settled on triple quote with a byte conversion?